ci/build_wheel_python.sh

```diff
@@ -30,7 +30,7 @@ PIP_CONSTRAINT="${PWD}/build-constraints.txt" \
 sccache --show-adv-stats
 
 mkdir -p final_dist
-python -m auditwheel repair -w final_dist dist/*
+python -m auditwheel repair --exclude librapids_logger.so -w final_dist dist/*
```

Comment:
Please write this with `EXCLUDE_ARGS` as we do in other packages. Example: https://github.com/rapidsai/raft/blob/140e974bee09a287406c84fde96d643909bdd5b8/ci/build_wheel.sh#L24

Reply:
Done.